frontend/src/metabase/dashboard/actions/sharing.ts

```diff
@@ -24,20 +31,23 @@ export const setSharing = (isSharing: boolean) => (dispatch: Dispatch) => {
   }
 };
 
-export const UPDATE_ENABLE_EMBEDDING =
-  "metabase/dashboard/UPDATE_ENABLE_EMBEDDING";
-export const updateEnableEmbedding = createAction(
-  UPDATE_ENABLE_EMBEDDING,
-  ({ id }: DashboardIdPayload, enable_embedding: boolean) =>
-    DashboardApi.update({ id, enable_embedding }),
-);
+export const PUBLISH_EMBEDDING = "metabase/dashboard/PUBLISH_EMBEDDING";
+export const publishEmbedding = createThunkAction(
+  PUBLISH_EMBEDDING,
+  ({ id, enable_embedding, embedding_params }: PublishEmbeddingPayload) =>
+    async () => {
+      const result = await DashboardApi.update({
+        id,
+        enable_embedding,
+        embedding_params,
+      });
```

Comment:
I was thinking about it and couldn't come up with any reason to exclude `embedding_parameters` when un-publishing an embedding.

Even more so, if I understand the logic correctly, we can change parameters and then un-publish, in which case we wouldn't even persist the changes? 

Might be wrong here but the less conditionals, the less confusing it is I suppose.